### PR TITLE
Removes conflict markers before diffing to improve dealing with conflict markers

### PR DESCRIPTION
**Pull request by hediet**

### CODE REVIEW

**Review by bpasero — CHANGES_REQUESTED** [review]
Compile

**Review by bpasero — CHANGES_REQUESTED** [review]
Tests

File: src/vs/workbench/contrib/mergeEditor/test/browser/projection.test.ts [hunk 1]
```diff
@@ -38,6 +38,7 @@ this.beakContainer.className = 'status-bar-beak-container';
 2
 3 // Beak Container
 4 this.beakContainer = document.createElement('div');
+䷅
```
Comment (bpasero) [inline src/vs/workbench/contrib/mergeEditor/test/browser/projection.test.ts:41]: Is that an intentional weird character change?

Comment (hediet) [reply src/vs/workbench/contrib/mergeEditor/test/browser/projection.test.ts:41]: Yes, see here: https://github.com/microsoft/vscode/pull/158283/files#diff-b85b02f00694da1c9fd7d4f71eed3156dd13189c0fdbd83771deea4f63654aacR130 .

**Review by bpasero — DISMISSED** [review]

**Review by bpasero — APPROVED** [review]

**Pull request merged**